docfx.json

```diff
@@ -140,21 +149,17 @@
           "**/includes/**"
         ],
         "src": "preview/docs-ref-conceptual",
```

Comment:
You can see many warning about missing ***.png. Can you update this path from `preview/docs-ref-conceptual` to `docs-ref-conceptual` to make resources properly published?